### PR TITLE
chore: Use version 0.0.7 of signed provisioning scripts

### PR DESCRIPTION
**Pull request by AbelHu**
chore: Use version 0.0.7 of signed provisioning scripts

Remove outdated 0.0.2

### CODE REVIEW

**Comment by mainred** [comment]
source of this change:  https://github.com/Azure/aks-engine/pull/4134
- feat: Updating kubelet/kube-proxy to run with as high priority processes (#4073)
- feat: Add WinDSR support (#4104)

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -100,9 +100,9 @@ function Get-FilesToCacheOnVHD {
             "https://github.com/microsoft/SDN/raw/master/Kubernetes/windows/helper.psm1",
             "https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1",
             "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg",
-            "https://acs-mirror.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.2.zip",
```
Comment (mainred) [inline vhdbuilder/packer/configure-windows-vhd.ps1:103]: I know 0.0.2 is old enough and not used in even aks-engine, but how do we define out-of-dated? can we remove `v0.0.3` sometime in the future?

**Review by mainred — APPROVED** [review]

File: vhdbuilder/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:103]: Good question. I think that we can keep 3 versions or at least 3 months for signed scripts. Let us try it and then refine the principal later.

**Comment by AbelHu** [comment]
@mainred v0.0.7 does not work with WinDSR so I closed this PR. Will open a new PR with new signed provisioning scripts.

**Pull request closed without merging**